frontend/packages/dev-console/integration-tests/support/commands/hooks.ts

```diff
@@ -1,9 +1,13 @@
+import { guidedTour } from '@console/cypress-integration-tests/views/guided-tour';
+
 before(() => {
-  const bridgePasswordIDP: string = Cypress.env('BRIDGE_HTPASSWD_IDP');
-  const bridgePasswordUsername: string = Cypress.env('BRIDGE_HTPASSWD_USERNAME');
-  const bridgePasswordPassword: string = Cypress.env('BRIDGE_KUBEADMIN_PASSWORD');
+  cy.exec(`../../../../create-user.sh`, { failOnNonZeroExit: false });
```

Comment:
Hey Sanket, this was the part we didn't test on Friday. But this "do not fail" (failOnNonZeroExit) makes me worried, so I run the tests manually and looked into the details.

TL;DR:

```suggestion
  cy.exec('../../../../contrib/create-user.sh');
```

How I come and verified this:

I inspected the exec command result in cypress and opened the browser log (F12) to see all the details when I clicked then on exec:

![Screenshot from 2023-07-02 19-32-22](https://github.com/openshift/console/assets/139310/e77f186b-f4bc-4572-b3bc-fde2739cec64)

I dropped failOnNonZeroExit to let the test fail and get the full error in the sidebar:

![Screenshot from 2023-07-02 19-33-49](https://github.com/openshift/console/assets/139310/08f7b511-bcd3-413e-8bb0-a73fa77bb677)

Because I didn't saw directly the error I changed the command to `pwd`:

![Screenshot from 2023-07-02 19-34-32](https://github.com/openshift/console/assets/139310/ad79234a-ef2d-4700-99c1-1c36002a0961)

Then I understand that the `contrib` folder was missing. After fixing the path I got:

![Screenshot from 2023-07-02 19-37-38](https://github.com/openshift/console/assets/139310/5b697dfb-93d9-45cd-97db-3065fc09c821)

So the create-user.sh script was started correctly. But it couldn't load the `htpasswd-secret.yaml` because the cwd was the integration-test folder instead of the console (root) folder.

For testing I removed the debug (`x`) option from `set -ex..`

![Screenshot from 2023-07-02 19-40-36](https://github.com/openshift/console/assets/139310/b806efa0-e591-4da8-804c-677c16c71bd3)

Then I tested the script from the terminal to reproduce the same error...

![Screenshot from 2023-07-02 19-42-48](https://github.com/openshift/console/assets/139310/6a58b7da-cea6-4922-9e19-ec4001c95845)

![Screenshot from 2023-07-02 19-43-00](https://github.com/openshift/console/assets/139310/149e9076-24b0-4d79-b780-66de4dc2c048)